system_tests/bigtable_happybase.py

```diff
@@ -618,6 +618,197 @@ def test_scan_timestamp(self):
         ])
 
 
+class TestTable_put(BaseTableTest):
+
+    def test_put(self):
+        value1 = 'value1'
+        value2 = 'value2'
+        row1_data = {COL1: value1, COL2: value2}
+
+        # Need to clean-up row1 after.
+        self.rows_to_delete.append(ROW_KEY1)
+        Config.TABLE.put(ROW_KEY1, row1_data)
+
+        row1 = Config.TABLE.row(ROW_KEY1)
+        self.assertEqual(row1, row1_data)
+
+        # Check again, but this time with timestamps.
+        row1 = Config.TABLE.row(ROW_KEY1, include_timestamp=True)
+        timestamp1 = row1[COL1][1]
+        timestamp2 = row1[COL2][1]
+        self.assertEqual(timestamp1, timestamp2)
+
+        row1_data_with_timestamps = {COL1: (value1, timestamp1),
+                                     COL2: (value2, timestamp2)}
+        self.assertEqual(row1, row1_data_with_timestamps)
+
+    def test_put_with_timestamp(self):
+        value1 = 'value1'
+        value2 = 'value2'
+        row1_data = {COL1: value1, COL2: value2}
+        ts = NOW_MILLIS
+
+        # Need to clean-up row1 after.
+        self.rows_to_delete.append(ROW_KEY1)
+        Config.TABLE.put(ROW_KEY1, row1_data, timestamp=ts)
+
+        # Check again, but this time with timestamps.
+        row1 = Config.TABLE.row(ROW_KEY1, include_timestamp=True)
+        row1_data_with_timestamps = {COL1: (value1, ts),
+                                     COL2: (value2, ts)}
+        self.assertEqual(row1, row1_data_with_timestamps)
+
+
+class TestTable_delete(BaseTableTest):
+
+    def test_delete(self):
+        table = Config.TABLE
+        value1 = 'value1'
+        value2 = 'value2'
+        row1_data = {COL1: value1, COL2: value2}
+
+        # Need to clean-up row1 after.
+        self.rows_to_delete.append(ROW_KEY1)
```

Comment:
They are RPC methods (so no 404s) and gRPC surfaces the errors as custom exceptions. But the tests run without error if that's what you're asking.